tidb-cloud/size-your-cluster.md

```diff
@@ -182,6 +182,31 @@ The supported node storage of different TiKV vCPUs is as follows:
 >
 > You cannot decrease the TiKV node storage after the cluster creation.
 
+### TiKV node storage types
+
+TiDB Cloud provides the following TiKV storage types for [TiDB Cloud Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-cloud-dedicated) clusters:
+
+#### Basic Type
+
+The Basic Type is a general-purpose storage type that provides lower performance than the Standard Type.
+
+The Basic Type is available for the following clusters:
+
+- Existing clusters that are created before April 1, 2025. 
+- New clusters that are created with versions earlier than v7.5.5, v8.1.2, or v8.5.0.
+
+#### Standard Type
+
+The Standard Type is ideal for most workloads due to its balanced performance and cost efficiency. It offers better performance compared to the Basic Type by reserving ample disk resources for Raft logs. This reduces the impact of Raft I/O on data disk I/O, resulting in improved read and write performance for TiKV.
+
+The Standard Type is available for new clusters created with versions v7.5.5, v8.1.2, or v8.5.0 or later.
+
+#### Performance Type（Coming soon）
+
+The Performance Type provides enhanced performance and stability, reducing P99 latency by up to 20% compared to the Basic Type. It allocates a dedicated high-performance disk for Raft logs, significantly boosting both performance and stability.
+
+This storage type will be available soon. 
+
```

Comment:
Avoid trying to document future features or products, even in innocuous ways. Don't pre-announce anything in documentation unless it has been approved by your legal counsel.

ref: https://developers.google.cn/style/future

```suggestion
```